sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/BranchPoint.java

```diff
@@ -25,13 +25,15 @@
   private final int offsetEnd;
   private final int startLine;
   private final int hits;
+  private final int path;
```

Comment:
it would be nice to add a comment for `path`

Reply:
(especially as there's no obvious link between the `BranchCoverage` class and OpenCover reports)

Reply:
`// Identifier for code path`